public/app/plugins/datasource/grafana-azure-monitor-datasource/variables.test.ts

```diff
@@ -5,6 +5,7 @@ import createMockDatasource from './__mocks__/datasource';
 import { invalidSubscriptionError } from './__mocks__/errors';
 import { AzureMonitorQuery, AzureQueryType } from './types';
 import { VariableSupport } from './variables';
+import { it } from 'test/lib/common';
```

Comment:
can you remove this line? It's causing the lint errors here: https://drone.grafana.net/grafana/grafana/57546/1/9 (`it` is automagically imported when using `jest`)

Reply:
Done!